### PR TITLE
Fix uploading of images

### PR DESCRIPTION
**Pull request by iryn34**
Add CASCADE to delete vote on event delete.
Upload images to amazon only after submitting edit forms (edit profile, edit team).
Delete old images from amazon during new images uploading.
Modify styles for event components.

### CODE REVIEW

File: eventually/static/src/containers/event/EventLink.js [hunk 1]
```diff
@@ -66,7 +71,16 @@ class EventLink extends React.Component {
 
     showAddress = () => {
         if (this.state.address) {
-            return 'Location: ' + this.state.address;
+            return (
+                <div>
+                    <IconLocationOn
+                        style={{float: 'left', color: '#777777'}}
```
Comment (VitalyiKhlivnyi) [inline eventually/static/src/containers/event/EventLink.js]: maybe, you shouldn't use inline style?

**Review by ashatna — DISMISSED** [review]

File: eventually/static/src/containers/event/EventLink.js (hunk 1 above)
Comment (iryn34) [reply eventually/static/src/containers/event/EventLink.js]: maybe

Comment (iryn34) [reply eventually/static/src/containers/event/EventLink.js]: done.

**Review by VitalyiKhlivnyi — DISMISSED** [review]

File: eventually/static/src/containers/profile/ProfileEdit.js [hunk 2]
```diff
@@ -99,30 +105,67 @@ export default class ProfileEdit extends React.Component {
...
+                });
+                this.hideLinearProgress();
+
+                putProfileService(
+                    this.state.id,
+                    this.state.firstName,
+                    this.state.middleName,
+                    this.state.lastName,
+                    this.state.hobby,
+                    this.state.imageName,
```
Comment (lhalam) [inline eventually/static/src/containers/profile/ProfileEdit.js]: use response.data['image_key']

**Review by VitalyiKhlivnyi — APPROVED** [review]

**Pull request merged**